libraries/stdlib/jvm-minimal-for-test/build.gradle.kts

```diff
@@ -89,5 +89,7 @@ publishing {
 
     repositories {
         maven("${rootProject.buildDir}/internal/repo")
+        mavenLocal()
```

Comment:
remove mavenLocal. That's what I was using to test locally but I'm not sure which public repo it should publish to. I guessed mavenCentral() but I am unsure

Reply:
@demiurg906 what is the publishing process of the compiler? Is adding the publishing plugin and a target repository sufficient for getting these modules published to maven central on a regular cadence? I have only tested using maven local repository via the gradle publishToLocalMaven command.

Reply:
I think it's better to ask @4u7

Reply:
When you say to replace the `internal publication setup` with those helpers, what do you mean by `internal publication setup`? Do you mean the publication tasks specified in build script for the jvm-minimal-for-test or are you referring to a parent build script?

Reply:
Is there an example build script somewhere that I can reference?

Reply:
```
Index: libraries/stdlib/jvm-minimal-for-test/build.gradle.kts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/libraries/stdlib/jvm-minimal-for-test/build.gradle.kts b/libraries/stdlib/jvm-minimal-for-test/build.gradle.kts
--- a/libraries/stdlib/jvm-minimal-for-test/build.gradle.kts	(revision b5b3d19f30d68a824d04bd7aca02f7e6a66078fc)
+++ b/libraries/stdlib/jvm-minimal-for-test/build.gradle.kts	(revision 4ac7cdbd8f62cf438bb49e5892187b5803fc36f7)
@@ -73,23 +73,12 @@
     }
 }
 
+publish()
+
 val jar = runtimeJar {
-    archiveFileName.set("kotlin-stdlib-minimal-for-test.jar")
     dependsOn(builtins)
     from(provider { zipTree(builtins.singleFile) }) { include("kotlin/**") }
 }
 
-publishing {
-    publications {
-        create<MavenPublication>("internal") {
-            artifactId = "kotlin-stdlib-minimal-for-test"
-            artifact(jar.get())
-        }
-    }
-
-    repositories {
-        maven("${rootProject.buildDir}/internal/repo")
-        mavenLocal()
-        mavenCentral()
-    }
-}
+sourcesJar()
+javadocJar()
Index: prepare/compiler/build.gradle.kts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/prepare/compiler/build.gradle.kts b/prepare/compiler/build.gradle.kts
--- a/prepare/compiler/build.gradle.kts	(revision b5b3d19f30d68a824d04bd7aca02f7e6a66078fc)
+++ b/prepare/compiler/build.gradle.kts	(revision 4ac7cdbd8f62cf438bb49e5892187b5803fc36f7)
@@ -181,7 +181,7 @@
         sources(project(":kotlin-stdlib-js", configuration = "distSources"))
         sources(project(":kotlin-reflect", configuration = "sources"))
 
-        distStdlibMinimalForTests(project(":kotlin-stdlib:jvm-minimal-for-test"))
+        distStdlibMinimalForTests(project(":kotlin-stdlib-minimal-for-test"))
 
         distJSContents(project(":kotlin-stdlib-js", configuration = "distJs"))
         distJSContents(project(":kotlin-test:kotlin-test-js", configuration = "distJs"))
Index: settings.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/settings.gradle b/settings.gradle
--- a/settings.gradle	(revision b5b3d19f30d68a824d04bd7aca02f7e6a66078fc)
+++ b/settings.gradle	(revision 4ac7cdbd8f62cf438bb49e5892187b5803fc36f7)
@@ -415,7 +415,7 @@
             ":kotlin-stdlib-jdk7",
             ":kotlin-stdlib-jdk8",
             ":kotlin-stdlib:samples",
-            ":kotlin-stdlib:jvm-minimal-for-test",
+            ":kotlin-stdlib-minimal-for-test",
             ":tools:binary-compatibility-validator",
             ":tools:kotlin-stdlib-gen",
 
@@ -430,10 +430,10 @@
     project(':kotlin-stdlib-js-ir').projectDir = "$rootDir/libraries/stdlib/js-ir" as File
     project(':kotlin-stdlib-wasm').projectDir = "$rootDir/libraries/stdlib/wasm" as File
     project(':kotlin-stdlib-js-ir-minimal-for-test').projectDir = "$rootDir/libraries/stdlib/js-ir-minimal-for-test" as File
+    project(':kotlin-stdlib-minimal-for-test').projectDir = "$rootDir/libraries/stdlib/jvm-minimal-for-test" as File
     project(':kotlin-stdlib-jdk7').projectDir = "$rootDir/libraries/stdlib/jdk7" as File
     project(':kotlin-stdlib-jdk8').projectDir = "$rootDir/libraries/stdlib/jdk8" as File
     project(':kotlin-stdlib:samples').projectDir = "$rootDir/libraries/stdlib/samples" as File
-    project(":kotlin-stdlib:jvm-minimal-for-test").projectDir = "$rootDir/libraries/stdlib/jvm-minimal-for-test" as File
 
     project(':tools:binary-compatibility-validator').projectDir = "$rootDir/libraries/tools/binary-compatibility-validator" as File
     project(':tools:kotlin-stdlib-gen').projectDir = "$rootDir/libraries/tools/kotlin-stdlib-gen" as File
```